python/cuml/test/test_random_forest.py

```diff
@@ -73,21 +73,31 @@ def test_rf_classification(datatype, split_algo, rows_sample, nrows,
                                    output_class=True,
                                    threshold=0.5,
                                    algo='auto')
+    fil_preds_proba = cuml_model.predict_proba(X_test,
+                                               output_class=True,
+                                               threshold=0.5,
+                                               algo='auto')
     cu_preds = cuml_model.predict(X_test, predict_model="CPU")
     fil_preds = np.reshape(fil_preds, np.shape(cu_preds))
     cuml_acc = accuracy_score(y_test, cu_preds)
     fil_acc = accuracy_score(y_test, fil_preds)
-
+    fil_calc_preds = []
+    for i in range(np.shape(fil_preds_proba)[0]):
+        if (fil_preds_proba[i, 0] >= fil_preds_proba[i, 1]):
+            fil_calc_preds.append(0)
```

Comment:
I think this needs more investigation to confirm that probabilities are in the same ballpark. Is fil_preds returning something else, like maybe missing the logistic transform at the end? Do you know why they are so different? Maybe try plotting fil class probs vs. sklearn class probs to see how they differ.

Reply:
(thinking about this now that it's AM) of course there should be no transform needed in these, so I'm not sure of the difference... definitely needs a look